glide.yaml

```diff
@@ -62,3 +62,5 @@ import:
   repo: https://github.com/openshift/gssapi.git
 - package: github.com/kylelemons/godebug
   version: fa7b53cdfc9105c70f134574002f406232921437
+- package: github.com/AlecAivazis/survey
```

Comment:
This is duplicated entry for `survay` We are already using `survey` package `v1` `"gopkg.in/AlecAivazis/survey.v1"` We should use just one version!